modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -0,0 +1,842 @@
+
+| Valid values
+| Duration.
+
+Zero and negative values result in lifetime not being checked.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.connection.max_lifetime[dbms.routing.driver.connection.max_lifetime]
+
+|===
+
+.dbms.routing.driver.connection_pool_acquisition_timeout
+[width="100%", cols="1s, 4a"]
+|===
+| Description
+|
+Maximum amount of time spent attempting to acquire a connection from the connection pool.
+This timeout only kicks in when all existing connections are being used and no new connections can be created because maximum connection pool size has been reached.
+Error is raised when connection can’t be acquired within configured time.
+
+| Valid values
+| Duration.
+
+Negative values are allowed and result in unlimited acquisition timeout.
+Value of `0` is allowed and results in no timeout and immediate failure when connection is unavailable.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.connection.pool.acquisition_timeout[dbms.routing.driver.connection.pool.acquisition_timeout]
+
+|===
+
+.dbms.routing.driver.connection.pool.max_size
+[width="100%", cols="1s, 4a"]
+|===
+
+| Description
+|
+Maximum total number of connections to be managed by a connection pool.
+The limit is enforced for a combination of a host and user.
+
+| Valid values
+| Integer.
+
+Negative values are allowed and result in unlimited pool.
+Value of `0` is not allowed.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.connection.pool.max_size[dbms.routing.driver.connection.pool.max_size]
+
+|===
+
+.dbms.routing.driver.logging.level
+[width="100%", cols="1s, 4a"]
+|===
+
+| Description
+| Sets level for driver internal logging.
+
+| Valid values
+| org.neo4j.logging.Level.
+
+One of `DEBUG`, `INFO`, `WARN`, `ERROR`, or `NONE`.
+
+| Default value
+| link:{neo4j-docs-base-uri}/operations-manual/{page-version}/configuration/configuration-settings#config_dbms.routing.driver.logging.level[dbms.routing.driver.logging.level]
+
+|===
+
+[NOTE]
+====
+If a transaction modifies a database alias, other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
+====
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+USER alice
+PASSWORD 'example_secret'
+DRIVER {
+  connection_timeout: duration({minutes: 1}),
+  connection_pool_max_size: 10
+}
+----
+
+When a database alias pointing to a remote database has been created, its details can be shown with the `SHOW ALIASES FOR DATABASE` command.
```

Comment:
I think you've switched most `pointing to` to `targeting` or similarly, should it also be done here? And if so it might be a good idea to do a proper search to see if there are other places missed updating